lighthouse-core/audits/deprecations.js

```diff
@@ -33,7 +33,7 @@ class Deprecations extends Audit {
     return {
       category: 'Deprecations',
       name: 'deprecations',
-      description: 'Site does not use deprecated APIs',
+      description: 'Avoids deprecated APIs',
```

Comment:
The complaint we've been getting (and which this PR tries to address) is that "not" in  `"X" Does not use....` is confusing. The "X" icon + "not" appear to folks as a double negative.

I also think there could be something better than "avoids" . Any suggestions?